lib/client.js

```diff
@@ -45,6 +45,7 @@ extend(Raven.prototype, {
     this.name = options.name || process.env.SENTRY_NAME || require('os').hostname();
     this.root = options.root || process.cwd();
     this.transport = options.transport || transports[this.dsn.protocol];
+    this.sendTimeout = options.sendTimeout || 1;
```

Comment:
Yea, there are a lot of situations where we're a little flexible/not-defensive but where it's fine as long as users aren't idiots. Since thus far there haven't been too many cases of that, I stick to the generally existing level of defensiveness; if we want to dial it up, there are lots of spots to add typeofs and bad-input-early-returns.